### PR TITLE
doc: provide translate of subscribe.md

### PR DESCRIPTION
**Pull request by msgui**
Subtask of #257, we could use a `TODO` tag to check the status: (**click** the button directly to mark it as done)

- [ ] update the EN community page
- [ ] update the CN community page & translation
- [x] update the EN page with [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list) in  [subscriber](https://hugegraph.apache.org/docs/contribution-guidelines/subscribe/) page
- [x] update the EN page with [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list) in  [subscriber-cn](https://hugegraph.apache.org/cn/docs/contribution-guidelines/subscribe/) page & translation
- [ ] enhance the content with `GPT/New Bing`
- [x] update the README info

**After：**
image

### CODE REVIEW

File: content/en/docs/contribution-guidelines/subscribe.md
Comment (imbajin) [inline content/en/docs/contribution-guidelines/subscribe.md:1]: Thanks, but we also need the `mailing-list` url for users to visit mails directly, some differ with [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list)

Comment (msgui) [reply content/en/docs/contribution-guidelines/subscribe.md:1]: Are you implying that the [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list) should align with the content of [subscribe.md](https://hugegraph.apache.org/cn/docs/contribution-guidelines/subscribe/#:~:text=%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1-,%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1,-TODO%3A%20translate%20) this? seem this quite is convenience.

image

**Comment by msgui** [comment]
> Thanks, but we also need the `mailing-list` **url** for users to visit mails directly, some differ with [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list)

Hi @imbajin ，
Are you implying that the [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list) should align with the content of [subscribe.md](https://hugegraph.apache.org/cn/docs/contribution-guidelines/subscribe/#:~:text=%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1-,%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1,-TODO%3A%20translate%20) this? seem this quite is convenience.

image

**Comment by imbajin** [comment]
> Are you implying that the [README](https://github.com/apache/incubator-hugegraph-doc#subscribe-the-mailing-list) should align with the content of [subscribe.md](https://hugegraph.apache.org/cn/docs/contribution-guidelines/subscribe/#:~:text=%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1-,%E8%AE%A2%E9%98%85%E7%A4%BE%E5%8C%BA%E9%82%AE%E7%AE%B1,-TODO%3A%20translate%20) this? seem this quite is convenience.

yep, and keep it clear (if the older page have some important info missed in the latest version, we could also add/enhance it)

**Comment by msgui** [comment]
Hello @imbajin ~，do the two pictures meet your expectations?
After：
cn
en

**Comment by imbajin** [comment]
> Hello @imbajin ~，do the two pictures meet your expectations? After： cn en

a little comment:

image

so as the CN doc

File: content/cn/docs/contribution-guidelines/subscribe.md [hunk 1]
```diff
@@ -1,29 +1,40 @@
 ---
-title: "订阅社区邮箱"
-linkTitle: "订阅社区邮箱"
+title: "订阅社区列表"
+linkTitle: "订阅社区列表"
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/subscribe.md:3]:
```suggestion
title: "订阅社区邮箱"
linkTitle: "订阅社区邮箱"
```

File: content/cn/docs/contribution-guidelines/subscribe.md [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
 weight: 2
 ---
 
-> TODO: translate this article to Chinese
+按照以下步骤订阅邮件列表：
 
-It is highly recommended to subscribe to the development mailing list to keep up-to-date with the community.
+- 通过您的电子邮件账户发送邮件至 [dev-subscribe@hugegraph.apache.org](mailto:dev-subscribe@hugegraph.apache.org)，然后您将收到一封确认邮件。
+- 回复确认邮件以确认您的订阅。然后，您将收到另一封确认邮件。
+- 现在您已经成为该邮件列表的订阅者。如果您有更多问题，只需发送邮件到邮件列表，有人会很快回复您。
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/subscribe.md:11]:
```suggestion
- 现在您已经成为该邮件列表的订阅者。如果您有更多问题，只需发送邮件到邮件列表，社区同学看到后会及时回复。
```

File: content/cn/docs/contribution-guidelines/subscribe.md [hunk 3]
```diff
@@ -1,29 +1,40 @@
...
 
-In the process of using HugeGraph, if you have any questions or ideas, suggestions, you can participate in the HugeGraph community building through the Apache mailing list. Sending a subscription email is also very simple, the steps are as follows:
+您可以随时订阅邮件列表。此外，您也可以直接浏览[历史邮件/所有邮件](https://lists.apache.org/list.html?dev@hugegraph.apache.org)（即使没有订阅列表）。
 
-1. Email dev-subscribe@hugegraph.apache.org with your own email address, subject and content are arbitrary.
+ 注意事项：
 
-2. Receive confirmation email and reply. After completing step 1, you will receive a confirmation email from dev-help@hugegraph.apache.org (if not received, please confirm whether the email is automatically classified as spam, promotion email, subscription email, etc.) . Then reply directly to the email, or click on the link in the email to reply quickly, the subject and content are arbitrary.
+ - 如果您没有收到确认邮件，请在24小时后再重试发送邮件。
+ - 在成功订阅邮件列表之前，请不要发送电子邮件到 **dev**（否则该邮件将被封禁）。
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/subscribe.md:19]:
```suggestion
 - 在成功订阅邮件列表之前，请勿发送电子邮件到 **dev**（未订阅发送的邮件将被被自动拦截）。
```

File: content/cn/docs/contribution-guidelines/subscribe.md [hunk 4]
```diff
@@ -1,29 +1,40 @@
...
 
-If you do not need to know what's going on with HugeGraph, you can unsubscribe from the mailing list.
+有关邮件订阅的更多信息，请参阅：
 
-Unsubscribe from the mailing list steps are as follows:
+- [https://lists.apache.org/list.html?dev@hugegraph.apache.org](https://lists.apache.org/list.html?dev@hugegraph.apache.org)
+- [http://apache.org/foundation/mailinglists.html#subscribing](http://apache.org/foundation/mailinglists.html#subscribing)
 
-1. Email dev-unsubscribe@hugegraph.apache.org with your subscribed email address, subject and content are arbitrary.
+# 退订邮件列表
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/subscribe.md:30]:
```suggestion
# 退订邮件列表 👋🏻 
```

**Review by imbajin — APPROVED** [review]

**Pull request merged**